src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers.ts

```diff
@@ -174,16 +159,54 @@ export class DatasetFlowTableHelpers {
         return "";
     }
 
-    public static waitingBlockText(startCondition: MaybeNull<FlowStartCondition>, datasetKind: DatasetKind): string {
+    public static durationBlockText(node: FlowSummaryDataFragment): string {
+        const descriptionTypename = node.description.__typename;
+        switch (descriptionTypename) {
+            case "FlowDescriptionDatasetPollingIngest":
+            case "FlowDescriptionDatasetPushIngest": {
+                const startCondition = node.startCondition as FlowStartConditionSchedule;
```

Comment:
who said polling ingest is only waiting on schedules? it could wait for throttling condition or executor

Reply:
Fixed